templates/telemetry/crash/crash.4.schema.json

```diff
@@ -57,10 +65,6 @@
               "description": "1, Optional, if set indicates that the crash had a memory report attached",
               "type": "string"
             },
-            "CrashTime": {
```

Comment:
Fortunately `CrashTime` and `crashTime` map to the same field in the generated schema.

Reply:
Actually, this field is completely separate from the other crashTime field.

https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/86956f9bf25a520c30da27b8a77b058f79c8a9f8/validation/telemetry/crash.4.sample.pass.json#L173-L183

Looking at the validation document, this is `payload.metadata.crash_time`, where-as the field you are adding is `payload.crash_time`. This field is [also mentioned in the firefox source docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/crash-ping.html). This should also be left since this would be a schema incompatible change.

Reply:
Oh right! Good catch! 💯